pkg/controller/mustgather/mustgather_controller_test.go

```diff
@@ -0,0 +1,98 @@
+package mustgather
+
+import (
+	"os"
+	"testing"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"github.com/redhat-cop/operator-utils/pkg/util"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/record"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	mustgatherv1alpha1 "github.com/openshift/must-gather-operator/pkg/apis/mustgather/v1alpha1"
+	"k8s.io/apimachinery/pkg/types"
+)
+func TestMustGatherController(t *testing.T) {
+	os.Setenv("JOB_TEMPLATE_FILE_NAME", "../../../build/templates/job.template.yaml")
+	
+	var cfg *rest.Config
+
+	mgObj := createMustGatherObject()
+	secObj := createMustGatherSecretObject()
+
+	objs := []runtime.Object {
+		mgObj,
+		secObj,
+	}
+
+	s := scheme.Scheme
+	s.AddKnownTypes(mustgatherv1alpha1.SchemeGroupVersion, mgObj)
+
+	cl := fake.NewFakeClientWithScheme(s, objs...)
+
+	eventRec := &record.FakeRecorder{}
+
+	recBase := util.NewReconcilerBase(cl, s, cfg, eventRec)
+
+	r := ReconcileMustGather{
+		ReconcilerBase: recBase,
+	}
+
+	req := reconcile.Request{
+		NamespacedName: types.NamespacedName{
+			Name: "test-must-gather",
+			Namespace: "openshift-must-gather-operator",
```

Comment:
shouldn't this be the name and namespace the mgObj has?
```suggestion
			Name: mgObj.Name,
			Namespace: mgObj.Namespace,
```

Reply:
+1